members/CusO1f14eac61548261742.yaml

```diff
@@ -0,0 +1,136 @@
+bioguide: CusO1f14eac61548261742
```

Comment:
another custom target added?

Reply:
this is odd, cuz this custom target was added ( and the other target WAL000038 deleted) by Tommy while I was out a couple of weeks ago. I merged these changes from my personal computer while wfh. I'm assuming that merging this PR to master might create a duplicate yaml for CusO1f14eac61548261742 right? If so, should I close this PR and merge again the new changes? (I'm back in the office today)

Reply:
try pulling down the latest contact congress master branch onto your machine, merging the master branch into develop, and then merging the updated develop branch into your `pull-and-fix-yamls` branch.

Reply:
Tried that with Anna. We don't understand why this PR is not updated with the latest commits, so going to close this one and open a new PR with the same changes.